Assets/_Scripts/Managers/CanvasManager.cs

```diff
@@ -0,0 +1,186 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+using UnityEditor;
+using UnityEngine.SceneManagement;
+using System.Runtime.CompilerServices;
+
+public class CanvasManager : MonoBehaviour
+{
+    [Header("Button")]
+    public Button startButton;
+    public Button mainMenuQuitButton;
+    public Button resumeButton;
+    public Button playAgainButton;
+    public Button mainMenuButton;
+    public Button gameOverQuitButton;
+    public Button pauseQuitButton;
+
+    [Header("Menus")]
+    public GameObject mainMenu;
+    public GameObject pauseMenu;
+    public GameObject gameOverMenu;
+
+    [Header("Text")]
+    public TMP_Text scoreText;
+    public TMP_Text HighScoreText;
```

Comment:
nit - public fields should all be Capitalized camel case. In either case it'd be best to make field names consistent (so either make all field names Capitalized camel case or change this line to `highScoreText`.